topics/proteomics/tutorials/metaquantome-function/tutorial.md

```diff
@@ -0,0 +1,424 @@
+> >
+> {: .solution}
+>
+{: .question}
+
+# **Create metaQuantome sample file**
+
+The create samples file module is used to generate the samples file input file for the metaQuantome workflow. This input file is used to specify the column names used for each experimental group. These column names are referenced when handling the input data and performing statistical analysis. 
+
+> ### {% icon hands_on %} Hands-on: Create an experimental Design file for sorting samples
+>
+> 1. {% tool [metaQuantome: create samples file](https://toolshed.g2.bx.psu.edu/view/galaxyp/metaquantome_sample/49a957fb353a) %} with the following parameters:
+>    - {% icon param-select %} *"Sample file creation method"*: `Manually specify experimental conditions and samples`
+>    - In *"Insert Samples"*:
+>        - *"1.Samples"*: 
+>            - *"Group Name"*: `T2`
+>               - *"Column"*: `T2_A1,T2_B1`
+>        - *"2.Samples"*: 
+>            - *"Group Name"*: `T4`
+>               - *"Column"*: `T4A_1,T4B_1`
+>        - *"3.Samples"*: 
+>            - *"Group Name"*: `T7`
+>               - *"Column"*: `T7A_1,T7B_1`
+>     
+> 2. Execute.
+>
+{: .hands_on}
+
+![MetaQuantome:Create-Sample](../../images/mQ-sample.png){: width="100%"}
+
+# **Run metaQuantome**
+
+
+## *metaQuantome: expand*
+
+The expand module is the first analysis step in the metaQuantome analysis workflow, and can be run to analyze differently expressed functions in the samples.
+In function mode, the following information is required apart from metaQuantome databases and samples file:
+- The name of the peptide column in the functional annotation file (` peptide`)
+- The name of the functional annotation column in the functional annotation file.(`go_term`)
+- The name of the peptide column in the Intensity file (`peptide`)
+
+> ### {% icon hands_on %} Hands-on: Run metaQuantome expand
+>
+> 1. {% tool [metaQuantome: expand](toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_expand/metaquantome_expand/2.0.0-0) %} with the following parameters:
+>    - {% icon param-file %} *"Database Archive File"*: `metaQuantome databases` (output of **metaQuantome: database** {% icon tool %})
+>    - {% icon param-file %} *"Samples file"*: `metaQuantome: create samples_file` (output of **metaQuantome: create samples file** {% icon tool %})
+>    - *"Mode"*: `Functional analysis`
+>        - {% icon param-file %} *"Functional file"*: `Function File` (Input dataset)
+>        - *"Functional file: peptide column name"*: `peptide`
+>        - *"Ontology"*: `Gene Ontology (GO terms)`
+>        - *"Functional column name"*: `go_term`
+>    - {% icon param-file %} *"Intensity file"*: `Intensity File` (Input dataset)
+>        - *"Intensity file: peptide column name"*: `peptide`
+> 2. Execute.
+>
+> 3. Rename as "metaQuantome expand".
```

Comment:
minor point, but for all these rename steps, we can add the pencil icon as a reminder to users how to do this:

```suggestion
> 3. **Rename** {% icon galaxy-pencil %} the output file to `metaQuantome expand`.
```